src/wmrc/config.py

```diff
@@ -24,9 +24,10 @@
     "from_address": "noreply@utah.gov",
     "to_addresses": [
         "jdadams@utah.gov",
```

Comment:
**suggestion**: remove comments and add `ugrc-developers@utah.gov`

Reply:
Good catch. I meant to ask if I could switch mine out for that but forgot.